cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+
+- Create a bug report
+- Create newcomer difficulty issue
+- Create a bear proposal issue
+- Create a documentation issue
+- Create low difficulty issue
+- Create medium difficulty issue
+- Create pull request that fixes a bug
+- Create pull request for bear proposal
+- Create pull request for a documentation issue
+- Create pull request for newcomer issue
+- Create pull request for low difficulty issue
+- Create pull request for medium difficulty issue
+- Gitmate errors on pull request
+- CI status of the pull request
+- Review comments on pull request
+- Number of questions asked in `coala/coala` room
+- Number of answers given in `coala/coala` room
+- Number of questions asked in `coala-bears` room
+- Number of  answers given in `coala-bears` room
+- Number of questions asked in `coala/corobo` room
+- Number of answers given in `coala/corobo` room
+- Number of questions asked in `coala/community` room
+- Number of answers given in `coala/community` room
+- Number of messages in coala other rooms
+- Make use of `corobo` on Gitter
+- Introduce `.coafile` in user own repository
+- Introduce `.coafile` in an organisation with 50 or more stars
+  for its repositories
+- Introduce CI in the users own repository to run coala using
+  Travis CI
+- Setup the Circle CI to the fork of coala repository
+- Setup Travis CI to the fork of coala repository
+- Setup Appveyor CI to the fork of coala repository
+
+### Points
+
+Points are a simple tool to reward positive contributions
+to coala, and they provide an excellent base upon which to
+build a gamified design. Now let's assign points to some
+of the tracked activities mentioned above:
+
+- Create Issue
+  - 20 points for type/bug
+  - 20 points for bear proposal
+  - 10 points for difficulty newcomer
+  - 20 points for difficulty low
+  - 30 points for difficulty medium
+- Merged Pull Request
+  - All the points will be calculated and assigned when the pull
+    request will be merged.
+  - If there are no Gitmate errors
+    - 20 points for bug fixes
+    - 20 points for writing a bear
+    - 10 points for difficulty newcomer
+    - 20 points for difficulty low
+    - 30 points for difficulty medium
+  - If Gitmate errors are b/w 1 to 5
+    - 15 points for bug fixes
+    - 15 points for writing a bear
+    - 7 points for difficulty newcomer
+    - 15 points for difficulty low
+    - 20 points for difficulty medium
+  - If Gitmate errors are b/w 5 to 10
+    - 10 points for bug fixes
+    - 10 points for writing a bear
+    - 5 points for difficulty newcomer
+    - 10 points for difficulty low
+    - 15 points for difficulty medium
+  - If Gitmate errors are b/w 10 and beyond
+    - No points are given
+- Activities related to introducing `.coafile`
+  - 10 points
+- Activities related to setting up CIs to the fork
+  - 10 points
+
+Note that these points allocations to the activities can be
+change during implementation, it is just an example to have
+a better understanding of this proposal.
```

Comment:
proposal -> design or system

(it stops being a proposal once this PR is merged)